src/main/drivers/serial_uart.c

```diff
@@ -71,6 +71,10 @@
     UART_BUFFER(UART_RX_BUFFER_ATTRIBUTE, n, R); struct dummy_s \
 /**/
 
+#ifdef USE_UART0
+UART_BUFFERS(0);
+#endif
+
 #ifdef USE_UART1
 UART_BUFFERS(1);
```

Comment:
```suggestion
    UART_BUFFERS(1);
```

Reply:
Following existing format - and remember these are macros that expand.